modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AssertionEncryptionConfiguration.java

```diff
@@ -0,0 +1,137 @@
+    }
+
+    @ApiModelProperty(example = "false")
+    @JsonProperty("enabled")
+    @Valid
+    public Boolean getEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(Boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    /**
+     *
+     **/
+    public AssertionEncryptionConfiguration assertionEncryptionAlgorithm(String assertionEncryptionAlgorithm) {
+
+        this.assertionEncryptionAlgorithm = assertionEncryptionAlgorithm;
+        return this;
+    }
+
+    @ApiModelProperty()
+    @JsonProperty("assertionEncryptionAlgorithm")
+    @Valid
+    public String getAssertionEncryptionAlgorithm() {
+        return assertionEncryptionAlgorithm;
+    }
+
+    public void setAssertionEncryptionAlgorithm(String assertionEncryptionAlgorithm) {
+        this.assertionEncryptionAlgorithm = assertionEncryptionAlgorithm;
+    }
+
+    /**
+     *
+     **/
+    public AssertionEncryptionConfiguration keyEncryptionAlgorithm(String keyEncryptionAlgorithm) {
+
+        this.keyEncryptionAlgorithm = keyEncryptionAlgorithm;
+        return this;
+    }
+
+    @ApiModelProperty()
+    @JsonProperty("keyEncryptionAlgorithm")
+    @Valid
+    public String getKeyEncryptionAlgorithm() {
+        return keyEncryptionAlgorithm;
+    }
+
+    public void setKeyEncryptionAlgorithm(String keyEncryptionAlgorithm) {
+        this.keyEncryptionAlgorithm = keyEncryptionAlgorithm;
+    }
+
+
+    @Override
+    public boolean equals(Object o) {
+
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        AssertionEncryptionConfiguration assertionEncryptionConfiguration = (AssertionEncryptionConfiguration) o;
+        return Objects.equals(this.enabled, assertionEncryptionConfiguration.enabled) &&
+                Objects.equals(this.assertionEncryptionAlgorithm,
+                        assertionEncryptionConfiguration.assertionEncryptionAlgorithm) &&
+                Objects.equals(this.keyEncryptionAlgorithm, assertionEncryptionConfiguration.keyEncryptionAlgorithm);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(enabled, assertionEncryptionAlgorithm, keyEncryptionAlgorithm);
+    }
+
+    @Override
+    public String toString() {
+
+        return "class AssertionEncryptionConfiguration {\n" +
+                "    enabled: " + toIndentedString(enabled) + "\n" +
+                "    assertionEncryptionAlgorithm: " + toIndentedString(assertionEncryptionAlgorithm) + "\n" +
+                "    keyEncryptionAlgorithm: " + toIndentedString(keyEncryptionAlgorithm) + "\n" +
+                "}";
+    }
+
+    /**
+     * Convert the given object to string with each line indented by 4 spaces
+     * (except the first line).
+     */
+    private String toIndentedString(Object o) {
+
+        if (o == null) {
+            return "null";
+        }
+        return o.toString();
+
+    }
+}
```

Comment:
Shall we add the missing new line at the end of file?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096